src/js/Tables/BasicTable.js

```diff
@@ -14,7 +14,7 @@ class BasicTable extends BaseTable {
 	 */
 	constructor(rootEl, sorter, opts = {}) {
 		super(rootEl, sorter, opts);
-		window.requestAnimationFrame(this.addSortButtons.bind(this));
+		window.setTimeout(this.addSortButtons.bind(this), 0);
```

Comment:
Why have we moved from rAF to setTimeout here?

Reply:
I wanted to avoid performing tasks within `requestAnimationFrame` which aren't updating  styles/DOM, like adding event listeners. `requestIdleCallback` is what I really want.